utils/payment-actions.js

```diff
@@ -186,3 +249,25 @@ export function insertFreeShippingCoupon() {
 export function goBackToShipping() {
   cy.get('#open-shipping').click()
 }
+
+export function payWithFoodVoucher(options = { withAddress: false }) {
+  selectFoodVoucherGroup()
+  waitLoad()
+  fillFoodVoucherInfo({ withAddress: options.withAddress, id: 0 })
+}
+
+export function combinePaymentMethods() {
+  cy.waitAndGet('#combine-payment-methods', 3000).click()
+}
+
+export function validateOrderPlaced(email) {
+  cy.url({ timeout: 120000 }).should('contain', '/orderPlaced')
+  cy.wait(2000)
+  cy.contains(email).should('be.visible')
+  cy.contains('Fernando Coelho').should('be.visible')
+  cy.contains('Rua Saint Roman 12').should('be.visible')
+  cy.contains('Copacabana').should('be.visible')
+  cy.contains('5521999999999').should('be.visible')
+  cy.contains('Cartão de crédito').should('be.visible')
+  cy.contains(`${CREDIT_CARD.FINAL}`).should('be.visible')
+}
```

Comment:
I think most payment tests could be refactored to use this, right?

To keep the code consistent, I believe that we could make this refactor or don't create this function right now. What do you think?